pkg/agent/controller/networkpolicy/packetin.go

```diff
@@ -117,12 +118,15 @@ func (c *Controller) storeDenyConnection(pktIn *ofctrl.PacketIn) error {
 		DestinationPort: packet.DestinationPort,
 		Protocol:        packet.IPProto,
 	}
+	// Retrieve string values of IP addresses to achieve deep copy of IP
```

Comment:
Thanks Anlan fixing this.
I think using copy() builtin function to do a deep copy for `net.IP` type will be more appropriate.
Ref: https://stackoverflow.com/questions/29732128/copy-net-ip-in-golang

Reply:
I once tried with `copy(tuple.SourceAddress, packet.SourceIP.To16())` but got `nil` as the value, maybe I was missing the initialization of tuple.SourceAddress before copy. Thanks! Let me try it again.